redbot/core/drivers/red_mongo.py

```diff
@@ -9,18 +9,24 @@
 
 
 def _initialize(**kwargs):
+    uri = kwargs["URI"]
```

Comment:
This breaks existing installations using Mongo since URI won't be a key in its storage details. I'd be happy for you to simply change this to `kwargs.get("URI", "mongodb")` as a solution.

Reply:
Change made, thanks